Gemfile.lock

```diff
@@ -0,0 +1,18 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    rack (1.6.4)
+    rack-cors (0.4.0)
+    roda (2.18.0)
+      rack
+
+PLATFORMS
+  ruby
```

Comment:
This will either be overwritten or even not working in production where jruby is used.